app/policies/task_policy.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class TaskPolicy < ApplicationPolicy
+  def task
+    @record
+  end
```

Comment:
I was shortly considering whether we should make this method private. However, since the `@record` variable is an `attr_reader`, the value is already accessibly publicly (on each `ApplicationPolicy` instance), so that this additional method in each policy shouldn't have any impact. Otherwise, one could argue that we don't want other callees to access it that way and hence make it private....
For simplicity, I would probably keep it.